luigi/format.py

```diff
@@ -510,13 +477,7 @@ def pipe_writer(self, output_pipe):
 SysNewLine = NewlineFormat()
 Gzip = GzipFormat()
 Bzip2 = Bzip2Format()
-MixedUnicodeBytes = MixedUnicodeBytesFormat()
```

Comment:
What is the replacement of this?

Reply:
@honnix 
no replacement as this format was introduced in #786 with comment
> Accept to write unicode in Files for compat  Only accept on python2 Fix #786

Reply:
Yeah it was added fo py2 compatibility, but could be abused for other purpose :( If for whatever reason, `lines` is a mixture of `str` and `bytes` and people decide to use this.

Reply:
I don't think people use `luigi.format.*` in their mundane lives outside `luigi`
do you think we should leave it here?

Reply:
I'm very sure people do and that is what these formatters for. I would suggest we leave it there and revamp the test case to reflect the "correct" usage: dealing with mixed of bytes and str. Wdyt?

Reply:
sounds good.
+ maybe we should add deprecation warning for that

Reply:
We don't need to deprecate it I think. It serves a use case and we have no replacement that we can recommend user to migrate to.